### PR TITLE
UIP-2272 Remove test utilities that belong in over_react_test

### PR DESCRIPTION
**Pull request by aaronlademann-wf**
> __Depends on https://github.com/Workiva/over_react_test/pull/11__

## Ultimate problem:
There were some test utilities in this library that were not exposed / available for consumers.

## How it was fixed:
[Move them to over_react_test](https://github.com/Workiva/over_react_test/pull/11)

## Testing suggestions:
Passing CI

## Areas of regression
Tests

---

__FYA:__ @jacehensley-wf @greglittlefield-wf @clairesarsam-wf

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Raven
<details>
<summary>Number of Findings: 0</summary>

</details>

File: tool/dev.dart [hunk 1]
```diff
@@ -36,8 +36,7 @@ main(List<String> args) async {
       // See https://github.com/Workiva/over_react/issues/36
       // 'chrome',
     ]
-    // Prevent test load timeouts on Smithy.
-    ..concurrency = 1
+    ..concurrency = 4
```
Comment (jacehensley-wf) [inline tool/dev.dart]: #nit 4 is the default, this can just be removed

File: pubspec.yaml [hunk 2]
```diff
@@ -37,3 +37,11 @@ transformers:
       $include: test/**_test{.*,}.dart
   # Reminder: dart2js should come after any other transformers that touch Dart code
   - $dart2js
+
+
+dependency_overrides:
+  over_react_test:
+    git:
+      url: git@github.com:aaronlademann-wf/over_react_test.git
```
Comment (jacehensley-wf) [inline pubspec.yaml]: travis doesn't support SSH urls :(

Comment (aaronlademann-wf) [inline pubspec.yaml]: Yeah, this will have to get removed before we merge anyway :man_shrugging:

**Comment by rmconsole-wf** [comment]
@aaronlademann-wf This pull request has merge conflicts, please resolve.

**Comment by aaronlademann-wf** [comment]
@jacehensley-wf @greglittlefield-wf @clairesarsam-wf this no longer has a dependent PR, and is ready for a final pass.

Now that over_react_test `1.1.0` has been released, over_react analyzer will fail until this is merged / released.

**Comment by aaronlademann-wf** [comment]
@jacehensley-wf @greglittlefield-wf @clairesarsam-wf this is ready for review.

**Comment by jacehensley-wf** [comment]
+1

**Comment by leviwith-wf** [comment]
QA +10
- Build successful
- tests pass
----
- [X] Testing instruction
- [X] Dev +1's
- [X] Unit tests created/updated
- [X] All unit tests pass
- [X] Rosie ran/Rosie comment displays expected info
- [X] Dependency Scan Clean

Merging.

**Pull request merged**